x-pack/packages/security-solution/navigation/src/landing_links/landing_links_images_cards.tsx

```diff
@@ -4,97 +4,77 @@
-    `,
-    titleContainer: css`
-      display: flex;
-      align-items: center;
-    `,
-    title: css`
-      color: ${euiTheme.colors.primaryText};
-    `,
-    description: css`
-      padding-top: ${euiTheme.size.xs};
-      max-width: 550px;
+    cardsContainer: css`
+      padding-top: 16px;
+      gap: ${euiTheme.size.s};
     `,
   };
 };
 
-const EuiCardWithLink = withLink(EuiCard);
-
 export const LandingLinksImageCards: React.FC<LandingLinksImagesProps> = React.memo(
   function LandingLinksImageCards({ items, urlState, onLinkClick }) {
+    const landingLinksAccordionId = useGeneratedHtmlId({ prefix: 'landingLinksAccordion' });
     const styles = useStyles();
+
     return (
-      <EuiFlexGroup direction="row" wrap data-test-subj="LandingImageCards">
-        {items.map((item) => {
-          const linkProps = getKibanaLinkProps({ item, urlState, onLinkClick });
-          const { id, landingImage, title, description, isBeta, betaOptions } = item;
-          return (
-            <EuiFlexItem
-              key={id}
-              data-test-subj="LandingImageCard-item"
-              grow={false}
-              css={styles.container}
+      <EuiPanel hasShadow={false} color="subdued" borderRadius="m" paddingSize="m">
+        <EuiAccordion
+          id={landingLinksAccordionId}
+          data-test-subj="LandingImageCards-accordion"
+          buttonContent={
+            <EuiFlexGroup
+              gutterSize="xs"
+              direction="row"
+              justifyContent="flexStart"
+              alignItems="center"
             >
-              <EuiCardWithLink
-                {...linkProps}
-                hasBorder
-                textAlign="left"
-                paddingSize="m"
-                css={styles.card}
-                image={
-                  landingImage && (
-                    <EuiImage
-                      data-test-subj="LandingImageCard-image"
-                      role="presentation"
-                      size={CARD_WIDTH}
-                      alt={title}
-                      src={landingImage}
-                    />
-                  )
-                }
-                title={
-                  <div css={styles.titleContainer}>
-                    <EuiTitle size="xs" css={styles.title}>
-                      <h2>{title}</h2>
-                    </EuiTitle>
-                    {isBeta && <BetaBadge text={betaOptions?.text} />}
-                  </div>
-                }
-                titleElement="span"
-                description={<span css={styles.description}>{description}</span>}
-              />
-            </EuiFlexItem>
-          );
-        })}
-      </EuiFlexGroup>
+              <EuiIcon type="logoSecurity" />
+              <EuiTitle size="xxs">
+                <h2>{i18n.LANDING_LINKS_ACCORDION_HEADER}</h2>
+              </EuiTitle>
+            </EuiFlexGroup>
+          }
+        >
+          <EuiFlexGrid css={styles.cardsContainer} columns={3}>
```

Comment:
Let's reuse the original layout with <EuiFlexGroup> and <EuiFlexItem> to achieve the requirement on wider screen.